moya/numbers.py

```diff
@@ -32,8 +32,13 @@ def csv_reader(fh, batch_size=1000):
     buf = []
     for row in csv.DictReader(fh):
         number = row['to']
+        
         if not is_number(number):
-            raise Exception("Line {number} doesn't look like a phone number")
+            continue
+        if number in deduplicate:
```

Comment:
create commandline option and pass through to activate this. may not be wanted by default.